integreat_cms/cms/linklists.py

```diff
@@ -18,7 +18,7 @@ class EventTranslationLinklist(Linklist):
     """
```

Comment:
In this file, the `html_fields` of `PageTranslationLinklist` as to be changed from `text` to `content`.